pkgs/applications/version-management/git-and-tools/git-when-merged/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, python }:
```

Comment:
I'm not sure we want to add new dependencies on `python` (which is `python2`), does this tool also work with `python3`?

Reply:
Works fine with Python3! Thanks for catching that.